ansible/roles/docker/templates/daemon.json.j2

```diff
@@ -0,0 +1,12 @@
+{
+  "hosts": ["unix://","tcp://0.0.0.0:2376"],
```

Comment:
This config should be parameterised as by default the port should not be opened top of the VM.

Reply:
This template has now been removed because docker fails to start with it.